public/javascripts/detailedResultsSlides.js

```diff
@@ -1,5 +1,61 @@
 /* eslint-disable no-unused-vars */
 
+function fetchSlideData(textHtml) {
+  fetch(
+    `/fetch-detailed-game-data?gameId=${
+      playSessionData.sessionData[slideIndex - 1].gameId
+    }`,
+    {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      data = JSON.parse(data);
+      let stores = "";
+      let metaScoreStyle = "";
+      if (data.metaScore >= 75) {
+        metaScoreStyle = "meta-green";
+      } else if (data.metaScore >= 50) {
+        metaScoreStyle = "meta-yellow";
+      } else {
+        metaScoreStyle = "meta-red";
+      }
+      data.stores.forEach((store) => {
+        stores += `<a href="${store.url}"><img class="store-icons" src="../../icons/store-icons/${store.store_id}.png"></a>`;
+      });
+      textHtml.innerHTML = `<div class="game-name user-guess-${
+        playSessionData.sessionData[slideIndex - 1].userGuess
+      }">${data.gameName}</div>
+        <div class="meta-score-wrapper"><div class="meta-score ${metaScoreStyle}">${
+        data.metaScore
+      }</div>
+        <p class="meta-score-text">Metacritic score</p></div>
+        <div class="game-details">
+        <div class="game-details-item">Genres:<p class="details-text">${
+          data.genres
+        }</p></div>
+        <div class="game-details-item">Developers:<p class="details-text">${
+          data.developers
+        }</p></div>
+        <div class="game-details-item">Platforms:<p class="details-text">${
+          data.platforms
+        }</p></div>
+        <div class="game-details-item">Stores:${stores}</div>
```

Comment:
If no store or website maybe you could not render these elements at all:

![image](https://github.com/DmitrijSidorkin/pixel-thing/assets/9388169/23a12a4a-f7d6-4e8b-a91d-f06af81ddb5a)